web/src/components/dashboard/SidebarCardsList.tsx

```diff
@@ -141,6 +127,55 @@ export const SidebarCardsList = (props: SidebarCardsListProps): ReactElement =>
     );
   };
 
+  const emptyForYouMessage = (): ReactNode => {
+    const showEmptyOpportunitiesMsg =
+      props.displayCertifications &&
+      props.displayFundings &&
+      props.certifications.length + props.fundings.length === 0;
+
+    const showCompleteRequiredTasksMsg =
+      !props.displayCertifications &&
+      !props.displayFundings &&
+      props.topCards.length + props.bottomCards.length === 0;
+
+    const isRemoteSellerWorker = (): boolean => {
+      if (business) {
+        const foreignBusinessType = determineForeignBusinessType(business.profileData.foreignBusinessTypeIds);
+        return foreignBusinessType === "REMOTE_SELLER" || foreignBusinessType === "REMOTE_WORKER";
+      }
+      return false;
+    };
+
+    const showEmptyForYouMessage =
+      showEmptyOpportunitiesMsg || showCompleteRequiredTasksMsg || isRemoteSellerWorker();
+
+    const getTopText = (): string => {
+      if (isRemoteSellerWorker()) return Config.dashboardDefaults.emptyOpportunitiesRemoteSellerWorkerText;
+      if (showCompleteRequiredTasksMsg) return Config.dashboardDefaults.completeRequiredTasksText;
+      if (showEmptyOpportunitiesMsg) return Config.dashboardDefaults.emptyOpportunitiesHeader;
+      return "";
+    };
+
+    const getBottomText = (): string => {
+      if (showEmptyOpportunitiesMsg) return Config.dashboardDefaults.emptyOpportunitiesText;
+      return "";
+    };
+
+    return (
+      <>
+        {showEmptyForYouMessage && (
+          <div className="fdc fac margin-y-3" data-testid="empty-for-you-message">
+            <div className="text-center margin-bottom-3">
+              <Content>{getTopText()}</Content>
```

Comment:
Pebble - since this is a child of `Content`, we should change the 3 widget types in the YAML file from string to markdown. The `getBottonText` fn would stay as a string.

Reply:
Another good callout